elyra/pipeline/processor_local.py

```diff
@@ -144,23 +144,24 @@ async def process(self, operation: Operation, filepath: str):
         pass
 
 
-class NotebookOperationProcessor(OperationProcessor):
-    _operation_name = 'execute-notebook-node'
+class FileOperationProcessor(OperationProcessor):
 
-    @property
-    def operation_name(self) -> str:
-        return self._operation_name
+    @abstractmethod
+    def _create_execute_command(self, filepath: str, cdw: str) -> str:
+        pass
 
     async def process(self, operation: Operation, filepath: str):
         if not os.path.isfile(filepath):
+            raise ValueError(f'Not a file: {filepath}')
+        if not os.path.exists(filepath):
             raise FileNotFoundError(f'Could not find {filepath}')
 
-        notebook_dir = os.path.dirname(filepath)
-        notebook_name = os.path.basename(filepath)
+        file_dir = os.path.dirname(filepath)
+        file_name = os.path.basename(filepath)
 
         self.log.debug(f'Processing: {filepath}')
 
-        argv = ['papermill', filepath, filepath, '--cwd', notebook_dir]
+        argv = self._create_execute_command(filepath, file_dir)
```

Comment:
I don't understand the utility of introducing this method.  The idea was to make the entire `process()` method common and only have the subclass provide an argv value.  This is just pushing the composition of the argv value to another location which isn't necessary if each subclass is already implementing the entire `process()` method anyway.